tests/views.py

```diff
@@ -16,8 +16,12 @@ def get(self, request, *args, **kwargs):
         return HttpResponse('<h1>Test content<h1>')
 
     def post(self, request, *args, **kwargs):
+        s_data = self.request.session.get('s_data')
         field, value = dict(request.POST).popitem()
-        return HttpResponse('<h1>{}: {}<h1>'.format(field, value[0]))
+        data = '<h1>{}: {}<h1>'.format(field, value[0])
+        if s_data:
```

Comment:
I wouldn't mix this up in `post`. Could you add this to the response in `get`, so that when you have session data, it should return `<h1>Test content<h1><p>Session data: s_data</p>` otherwise just return `<h1>Test content<h1>`.